Abidsyed25/ScrapQuest
# Enhance Contact Page Responsiveness for All Devices

**Made changes for #64** 

**Description:**
This PR addresses the responsiveness of the contact page, ensuring it is compatible and visually appealing across all devices, including mobile phones and tablets. Below are the changes made to achieve this:

**Changes Made:**

1. Responsive Padding: Adjusted the padding of the main container and elements using Tailwind CSS responsive utility classes (p-4, sm:p-20, etc.) to ensure adequate spacing on different screen sizes.
2. Responsive Text Sizes: Implemented responsive text size classes (text-xl, sm:text-2xl, etc.) to ensure the text scales appropriately and remains readable on smaller screens.
3. Form Container Width: Ensured the form container utilizes w-full and other responsive classes to adjust its width according to the screen size, providing a better user experience on mobile devices.


**Screenshots:**
![image](https://github.com/Abidsyed25/ScrapQuest/assets/117517206/dee25e5a-a71c-4b25-88c4-341b65c512ce)

**Before:**
The contact form elements were not scaling appropriately on smaller screens, leading to a poor user experience.
**After:**
The contact form is now fully responsive, providing a seamless experience on devices of various sizes, as shown in the above attached screenshot.

**How to Test:**

1. Pull the latest changes from the branch.
2. Navigate to the Contact page.
3. Resize the browser window or use the device toolbar to simulate different screen sizes.
4. Verify that all elements scale appropriately and maintain a consistent layout.
